packages/extension-link/src/link.ts

```diff
@@ -75,9 +75,21 @@ export interface LinkOptions {
   /**
    * A validation function that modifies link verification for the auto linker.
    * @param url - The url to be validated.
+   * @param ctx - An object containing:
+   *              - `defaultValidate`: A function that performs the default URL validation.
+   *              - `protocols`: An array of allowed protocols for the URL (e.g., "http", "https").
+   *              - `defaultProtocol`: A string that represents the default protocol (e.g. 'http')
```

Comment:
You can do jsdoc on each value like this:

```
{
/** comment **/
defaultValidate: (url: string) => boolean;
/** another **/
protocols: Array<LinkProtocolOptions | string>;
/** and again **/
defaultProtocol: string
}
```

This allows typescript to show you what each key of that object does